Provider/src/main/java/com/spotify/confidence/EventSenderUploader.kt

```diff
@@ -52,11 +52,13 @@ internal class EventSenderUploaderImpl(
     }
 
     override suspend fun upload(events: EventBatchRequest): Boolean = withContext(dispatcher) {
+        Event
```

Comment:
👀

Reply:
Wha.... What happened there?? And how is this still compiling :D